src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Toolbar.js

```diff
@@ -110,15 +111,16 @@ export default class Toolbar extends React.Component<*> {
                 }
                 {!!Initializer.initializedTranslationsLocale &&
                     <ToolbarComponent.Snackbar
-                        onClick={onNavigationButtonClick}
+                        onClick={navigationPinned ? undefined : onNavigationButtonClick}
                         type="success"
                         visible={this.toolbarStore.showSuccess}
                     />
                 }
                 <ToolbarComponent.Controls grow={true}>
                     {!!onNavigationButtonClick &&
                     <ToolbarComponent.Button
-                        icon={navigationOpen ? 'su-times' : 'su-bars'}
+                        disabled={navigationPinned}
+                        icon={navigationPinned ? 'su-sulu' : (navigationOpen ? 'su-times' : 'su-bars')}
```

Comment:
Did you discuss that the Sulu Icon is shown if the navigation is pinned with somebody? Can understand that it looks a bit boring without the blue button, but on the other side it also takes quite some place from the toolbar, which might be better used in different ways (e.g. showing text for the other buttons instead of collapsing).